runtime/autoload/provider/perl.vim

```diff
@@ -11,6 +11,12 @@ function! provider#perl#Detect() abort
     return ''
   endif
 
+  " if perl is available, make sure we have 5.20+
+  call system([prog, '-e', 'use v5.20'])
```

Comment:
Why is it 5.20+ here but the documentation requires 5.22+?

Reply:
Yep, thats not right. I've fixed it.